RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py

```diff
@@ -19,8 +19,8 @@
 # ECAL multifit running on GPU
 from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitProducerGPU_cfi import ecalUncalibRecHitProducerGPU as _ecalUncalibRecHitProducerGPU
 ecalMultiFitUncalibRecHitGPU = _ecalUncalibRecHitProducerGPU.clone(
-  digisLabelEB = cms.InputTag('ecalDigisGPU', 'ebDigisGPU'),
-  digisLabelEE = cms.InputTag('ecalDigisGPU', 'eeDigisGPU'),
+  digisLabelEB = cms.InputTag('ecalDigisGPU', 'ebDigis'),
```

Comment:
Remove `cms.InputTag` as well here as done in `HLTrigger/Configuration/python/customizeHLTforPatatrack.py`?

Reply:
Ah well it is already merged. Did not see that.

Reply:
thanks for the comment, I can add this later anyway